### ISSUE TITLE
Extra space is inserted on links that span more than one line in po file

### ISSUE THREAD

**Issue by EuAndreh** [mondeja/mdpo#147]
When a big link exists, md2po will break that link across several lines on the
generated po file.  However it doesn't glue it back together correctly on the
translated file.

This makes 
```
[x](https://example.com/a-big-link)
```

become
```
[x](https://example.com/a-big -link)
```

The link itself doesn't need to actually be very big, it only needs to be
positioned at the end of the msgid line.

Sample interactive session:
```
$ pushd `mktemp -d`
/tmp/tmp.BMW6tD7H0q ~
$ echo '[a link name](https://example.com/a-big-link-with-a-very-very-long-and-big-url-name-on-it)' > sample.md
$ md2po sample.md --po-filepath l.po --save
#
msgid ""
msgstr ""

#: sample.md:block 1 (paragraph)
msgid ""
"[a link name](https://example.com/a-big-link-with-a-very-very-long-and-big-"
"url-name-on-it)"
msgstr ""

$ po2md sample.md --pofiles l.po --save sample.l.md
$ cat sample.l.md
[a link name](https://example.com/a-big-link-with-a-very-very-long-and-big-url-
name-on-it)
$ pandoc -i sample.md -w html
<p><a href="https://example.com/a-big-link-with-a-very-very-long-and-big-url-name-on-it">a link name</a></p>
$ pandoc -i sample.l.md -w html
<p><a href="https://example.com/a-big-link-with-a-very-very-long-and-big-url-%20name-on-it">a link name</a></p>
$ md2po --version
.md2po-real 0.3.53
```

Extracted steps to reproduce:
```
pushd `mktemp -d`
echo '[a link name](https://example.com/a-big-link-with-a-very-very-long-and-big-url-name-on-it)' > sample.md
md2po sample.md --po-filepath l.po --save
po2md sample.md --pofiles l.po --save sample.l.md
cat sample.l.md
pandoc -i sample.md -w html
pandoc -i sample.l.md -w html
md2po --version
```

The link in the translated HTML generated by pandoc on this case doesn't represent the original.

**Comment by mondeja** [comment]
Many thanks for the detailed report! It's fixed in [v0.3.54](https://github.com/mondeja/mdpo/releases/tag/v0.3.53).

**Comment by EuAndreh** [comment]
Thanks for addressing the issue!

I came back to check its status now, and I'm gifted
with it being already fixed :)
